### DISCUSSION TITLE
Free Rides

### DISCUSSION THREAD

**Discussion by kenlpeters** [philosowaffle/peloton-to-garmin#104] [Q&A]
So this week I need to get my 40K badge for cycling.  Based on history that will take me a little more 90 mins.  So there is not Peloton class long enough to get the 40K in a single activity.  So looks like I am going to do a free ride.  Do free rides sync just like regular classes?

**Comment by GregChung-IBFL** [comment]
I have not had any issues syncing "Just Ride" activities to Garmin.  However, the Garmin challenges I think differentiate between regular cycling and indoor cycling?  If you upload the ride, I think you're credited for the (outdoor) badge (ie the 40K) but not the indoor badge?

**Reply by kenlpeters** [reply]
Thanks for the information.  This adds some comfort.  I did not want to do 90 min on the bike and not get the damn badge.  I agree  with you.  I think the 40K badge this up coming weekend is cycling in general.  I have got it before using my Echelon, this will be my first 40K badge with a Peloton.

**Reply by GregChung-IBFL** [reply]
For extra comfort, do a 5 minute free ride and sync that first :)

**Reply by kenlpeters** [reply]
I was thinking about doing that.  I figured I would ask before trying it myself because if the answer was "no" I had to research another solution.  Sounds like I am in the clear.  I will do a 5 minute free ride tomorrow.
